src/rgw/rgw_rest.cc

```diff
@@ -574,6 +574,7 @@ void abort_early(struct req_state *s, RGWOp *op, int err_no)
 
 void dump_continue(struct req_state *s)
 {
+  perfcounter->inc(l_rgw_http_status_1xx);
```

Comment:
I found that 100 Continue is not a final status, RGW first sends 100 then the actual HTTP status for requests with Expect: 100 continue header. So one request generates two HTTP status.


vagrant@vagrant-debian-wheezy:~/ceph/src$ sudo httpry -i lo tcp port 8000          httpry version 0.1.7 -- HTTP logging and information retrieval tool
Copyright (c) 2005-2012 Jason Bittel <jason.bittel@gmail.com>
----------------------------
Hash buckets:       64
Nodes inserted:     10
Buckets in use:     10
Hash collisions:    0
Longest hash chain: 1
----------------------------
Starting capture on lo interface

2016-08-07 00:21:57	127.0.0.1	127.0.0.1	>	PUT	localhost:8000	/prashant/hello	HTTP/1.1	-	-
2016-08-07 00:21:57	127.0.0.1	127.0.0.1	<	-	-	-	HTTP/1.1	100	CONTINUE
2016-08-07 00:21:59	127.0.0.1	127.0.0.1	<	-	-	-	HTTP/1.1	200	OK